integration/add_ats.sh

```diff
@@ -0,0 +1,68 @@
+#! /bin/bash
+# desc: generates new ats files for each test based off input file structure
+
+ORIG_DIR=$(pwd) # should be ./serac_repo/tests/integration
+INPUT_DIR="${ORIG_DIR}/../../data/input_files/tests/"
+
+# for each type,
+TYPES=(solid thermal_solid thermal_conduction)
+
+cd ${INPUT_DIR}
+
+for _type in ${TYPES[@]} ; do
+
+    cd "./${_type}"
+
+    # for each test in directory "t"
+    for t in $(ls --color=no) ; do
+
+        # remove .lua from file name
+        t=$(echo $t | awk -F "." '{print $1}')
+
+        if [ $t == "dyn_amgx_solve" ] ||
```

Comment:
Is there a reason these are being skipped?

Reply:
Nevermind, just saw the issues. AMGx only works on GPU machines. I will investigate the static reaction one.

Reply:
the static reaction one times out when i run it, which is why I skip that one. https://github.com/LLNL/serac/issues/654

Reply:
Yeah, that shouldn't be timing out. I'll see what's up.